network-api/networkapi/wagtailpages/models.py

```diff
@@ -54,30 +58,31 @@
 
 __all__ = [
     base_fields,
-    FoundationMetadataPageMixin,
-    Signup,
-    ModularPage,
-    MiniSiteNameSpace,
-    PrimaryPage,
-    IndexPage,
-    NewsPage,
-    InitiativesPage,
-    ParticipatePage2,
-    Styleguide,
-    Homepage,
-    RedirectingPage,
     BanneredCampaignPage,
-    CampaignPage,
-    OpportunityPage,
+    BlogIndexPage,
```

Comment:
alphabetical sort. Import order is _very important_ but export ordering is irrelevant, so sorting is nice.